src/core-decorators.ts

```diff
@@ -8,20 +8,19 @@
 export { default as override } from './override';
 export { default as deprecate, default as deprecated } from './deprecate';
 export { default as suppressWarnings } from './suppress-warnings';
-export { default as memoize } from './memoize';
 export { default as autobind } from './autobind';
 export { default as readonly } from './readonly';
 export { default as enumerable } from './enumerable';
 export { default as nonenumerable } from './nonenumerable';
 export { default as nonconfigurable } from './nonconfigurable';
-export { default as debounce } from './debounce';
-export { default as throttle } from './throttle';
 export { default as decorate } from './decorate';
-export { default as mixin, default as mixins } from './mixin';
 export { default as lazyInitialize } from './lazy-initialize';
 export { default as time } from './time';
 export { default as extendDescriptor } from './extendDescriptor';
 export { default as profile } from './profile';
 
+// Exported for testing purposes
+export  {default as defaultConsole} from './defaultConsole';
```

Comment:
P.S.   I'd like to add tslint too, currently these escaped eslint because of the file extension.